R/defaultThemes.R

```diff
@@ -139,6 +139,8 @@ initDefaultTheme <- function(){
   theme$subtitle_adj_r <- .9
   theme$legend_intersp_x <- 1
   theme$legend_intersp_y <- 1
+  theme$y_grid_count_strict <- TRUE # do not allow extra ticks to be attached by y_tick_margin (which could lead to y_grid_count being violated)
+  theme$y_tick_margin <- 0.5        # at least 50% of top/bottom tick range must be empty if y_grid_count_strict
```

Comment:
good idea to introduce another parameter as opposed of a fixed .5

Reply:
@HomoCodens: 

I checked out *your* branch, tested it. And yes, what you said ist true on your branch: left and right ticks don't correspond and therefore only left ticks go with the horizontal grid. Even in the default theme. 

but it's not true for the original implementation. checkout master to get:

```
tsplot(KOF$kofbarometer,
       tsr = KOF$reference,auto_legend=F)
```

![image](https://user-images.githubusercontent.com/1382079/35216030-7cd08c90-ff66-11e7-80b4-09410c6cb65e.png)